modules/dop/endpoints/projectpipelinefiletree.go

```diff
@@ -65,7 +68,13 @@ func (e *Endpoints) GetFileTreeNode(ctx context.Context, r *http.Request, vars m
 	if err := e.queryStringDecoder.Decode(&req, r.URL.Query()); err != nil {
 		return apierrors.ErrListFileTreeNodes.InvalidParameter(err).ToResp(), nil
 	}
-	req.Inode = vars["inode"]
+	inode, err := url.QueryUnescape(vars["inode"])
+	if err != nil {
+		logrus.Error(apierrors.ErrGetFileTreeNode.InvalidParameter("QueryUnescape inode"))
```

Comment:
use `logrus.Errorf` and print invalid inode

Reply:
done